src/css/how-to-place.css

```
@@ -0,0 +1,88 @@
+#content-wrapper {
+    display: flex;
+    flex-direction: column;
+    margin: 2vh 1vw;
+}
+
+#title-wrapper {
+    margin: 1vh 1vw;
+    padding: 4vh 2vw;
+    border-style: solid;
+    border-width: 0.1vh;
+    border-color: #3f3f3f;
+    border-radius: 0.5rem;
+    background-color: #dddddd;
+    text-align:center;
+}
+
+h1 {
+    font-size: 2rem;
+}
+

```

This should help set the header sizes.

```suggestion

#content-wrapper h1 {
    font-size: 2rem;
}

#content-wrapper h2 {
    font-size: 1.6rem;
}

```